test/e2e/specs/all-features-confirmation/recordset/presentation.spec.js

```diff
@@ -33,56 +37,134 @@ var testParams = {
                 luxurious: "true",
                 json_col: JSON.stringify({"name":"testing_json"},undefined,2),
                 json_col_with_markdown: "Status is: “delivered”",
+                category: "Resort",
+                type_of_facilities: "Luxury",
                 count_image_id: "1",
                 count_distinct_image_id: "1",
                 min_image_id: "3001",
                 max_image_id: "3001"
             },
             {
                 id: 2002,
+                no_of_rooms: '23',
                 title: "Sherathon Hotel",
-                website: "http://www.starwoodhotels.com/sheraton/index.html",
+                website: "http://www.starwoodhotels.com/sheraton/index.html", 
                 rating: "4.3000",
                 summary: "Sherathon Hotels is an international hotel company with more than 990 locations in 73 countries. The first Radisson Hotel was built in 1909 in Minneapolis, Minnesota, US. It is named after the 17th-century French explorer Pierre-Esprit Radisson.",
                 opened_on: "2008-12-09 00:00:00",
                 luxurious: "true",
                 json_col: JSON.stringify(null,undefined,2),
-                json_col_with_markdown: "Status is: “delivered”",
+                json_col_with_markdown: "Status is: “delivered”",              
```

Comment:
It looks like you added a few extra whitespaces to the end of this line. After highlighting it:
![image](https://user-images.githubusercontent.com/2932600/41004934-6eee9c6a-68d0-11e8-81ff-9fa83d5659dc.png)

Is there an option in your editor to remove trailing whitespace on save? This will prevent git from flagging some lines as changed when it was only whitespace.